tutorials/plot_visualize_raw.py

```diff
@@ -105,6 +105,15 @@
 raw.add_proj(projs)
 raw.plot_projs_topomap()
 
+###############################################################################
+# Note that the projections in ``raw.info['projs']`` can be visualized using
+# ``raw.plot_projs_topomap`` or calling
+# :func:`proj.plot_topomap <mne.io.proj.Projection.plot_topomap>`
```

Comment:
This will not link properly
```
/home/circleci/project/doc/auto_tutorials/plot_visualize_raw.rst:227: WARNING: py:func reference target not found: mne.io.proj.Projection.plot_topomap
```
See where we have it in `python_reference.rst` -- it is just [mne.Projection](https://9990-1301584-gh.circle-artifacts.com/0/html/python_reference.html) so here you do
```
:func:`proj.plot_topomap <mne.Projection.plot_topomap>`
```
and again you can check locally that you see this Sphinx error disappear